src/fidesctl/core/system.py

```diff
@@ -422,10 +305,10 @@ def scan_system_aws(
     )
 
 
-def _check_boto3_import() -> None:
+def _check_aws_connector_import() -> None:
```

Comment:
would this also be better to have in a shared space alongside the connectors possibly?